src/redux/actions/projects/loadProjects.js

```diff
@@ -0,0 +1,36 @@
+import fetchAPI from '../../../utils/fetchAPI';
+import { PROJECTS_ERROR, PROJECTS_LOAD } from '../../actionTypes/projects';
+import pushNotificationMessage from '../notifications';
+import messages from '../../../components/notification/messages';
+import loadSamples from '../samples/loadSamples';
+
+const loadProjects = () => async (dispatch) => {
+  try {
+    const response = await fetchAPI('/v1/projects');
```

Comment:
here is the place where you should dispatch a `PROJECTS_LOADING` action. You should also write the relevant reducer to handle this state. For an example of how to do it, see `loadCellSet.js` and its relevant reducers and action types, it is a very similar use case.